pysb/importers/bngl.py

```diff
@@ -0,0 +1,319 @@
+from pysb.core import MonomerPattern, ComplexPattern, RuleExpression, \
+    ReactionPattern, ANY, WILD
+from pysb.builder import Builder
+from pysb.bng import BngConsole
+import xml.etree.ElementTree
+import re
+from sympy.parsing.sympy_parser import parse_expr
+import warnings
+
+
+def _ns(tag_string):
+    """
+    Shorthand to apply XML namespace
+    """
+    return tag_string.format('{http://www.sbml.org/sbml/level3}')
+
+
+class BnglImportError(Exception):
+    pass
+
+
+class BnglBuilder(Builder):
+    """
+    Class for assembling a Model from a .bngl file. See
```

Comment:
First line of a docstring is supposed to be just one line. The details go in the body, after a blank line. Also I think "Class for..." is redundant. Perhaps just "Assemble a Model from a .bngl file".